HCL.IdentityServer.API.BLL/Services/RegistrationService.cs

```diff
@@ -54,6 +53,7 @@ public async Task<BaseResponse<AuthDTO>> Authenticate(AccountDTO DTO)
             if (account.Data == null ||
                 !VerifyPasswordHash(DTO.Password, Convert.FromBase64String(account.Data.Password), Convert.FromBase64String(account.Data.Salt)))
             {
+
```

Comment:
Code from 68 line to 107. It is better to create separate classes for them. Something like tokenService and HashHelper(bll)